client/src/pages/SearchPage.js

```diff
@@ -0,0 +1,129 @@
+import { Link } from 'react-router-dom'
+import styled from 'styled-components'
+import Poster from '../components/Poster'
+import useSearch from '../hooks/useSearch'
+
+export default function SearchPage() {
+  const { query, results, handleSearch } = useSearch()
+
+  return (
+    <Wrapper>
+      <StyledLabel>
+        Suche:
+        <StyledInput
+          type="text"
+          placeholder="Was willst du finden?"
+          onChange={handleSearch}
+          onKeyDown={event =>
+            (event.key === 'Enter' || event.key === 'Escape') &&
+            event.target.blur()
+          }
+        />
+      </StyledLabel>
+      {results !== null && (
```

Comment:
Shouldn't that be the same as:
```suggestion
      {results && (
```
If `results === null` it's falsy. And only if it's filled with something, it is truthy and therefore the block will be rendered.